whipper/command/cd.py

```diff
@@ -341,7 +350,19 @@ def doCommand(self):
         else:
             logger.info("creating output directory %s", dirname)
             os.makedirs(dirname)
-
+        self.coverArtPath = None
+        if (self.options.fetch_cover_art in {"embed", "complete"} and
+                importlib.util.find_spec("PIL") is None):
+            logger.warning("the cover art option '%s' won't be honored "
+                           "because the 'pillow' module isn't available",
+                           self.options.fetch_cover_art)
+        elif self.options.fetch_cover_art in {"file", "embed", "complete"}:
+            self.coverArtPath = self.program.getCoverArt(
+                                    dirname,
+                                    self.options.fetch_cover_art,
```

Comment:
`getCoverArt()` signature suggests that the second parameter is "path", which seems different from what is being passed here. Did you test this in practice?

Reply:
@Freso , the first parameter is actually `self` so `dirname` is `path` ?

Reply:
Ah, you’re right, in that case there are only 2 parameters for `getCoverArt()`, and this call provides it 3. :)

Reply:
Wait what, i remember clearly i have removed it.

Reply:
My fault: error during rewrite :rofl:

Reply:
> My fault: error during rewrite 🤣

No problem, it's your bad memory i guess XD .

Reply:
Should be fixed now!